libs/models/src/lib/sockets/race.ts

```diff
@@ -1,8 +1,20 @@
-import { PlayerWithLogs } from './player';
+import { z } from 'zod';
+import { PlayerLogsCollection } from './playerLog';
 
+// ==== Primary Schemas ==== //
+const raceIdSchema = z.custom<`T:${string}-R:${string}`>(id =>
+  /^T:[a-zA-Z0-9]{8}-R:[a-zA-Z0-9]{3}$/.test(id as string),
+);
+
+// ==== Types ==== //
+// Race id template literal
+export type raceId = z.input<typeof raceIdSchema>;
```

Comment:
Can we have this renamed to `RaceId`?

Reply:
Types need to be in PascalCase

Reply:
fixed de1f40717ab164f66408e3bf2acbe282ff4ce7e7